structure/templates/structure_browser.html

```diff
@@ -12,6 +12,30 @@
     <link href="{% static 'home/css/sequenceviewer.css' %}" rel="stylesheet">
     <link href="{% static 'home/css/modal.css' %}" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="{% static 'home/css/jquery.powertip.css' %}" />
+    <style>
+      /*Structure preview in tooltip*/
+      pre {

```

Comment:
Small note: it can be dangerous to change the formatting of an HTML element as it will of course impact all occurrences of the element + it is cleaner not to have CSS embedded in the templates.